apps/examples/echo/rpmsg-ping.c

```diff
@@ -44,7 +44,7 @@ static int rpmsg_endpoint_cb(struct rpmsg_endpoint *ept, void *data, size_t len,
 	(void)ept;
 	(void)src;
 	(void)priv;
-	LPRINTF(" received payload number %lu of size %lu \r\n",
+	LPRINTF(" received payload number %lu of size %lu \n",
```

Comment:
In some system, if you don't use "\r", the next new line doesn't start from the beginning. make the output hard to read.
Prefer not to remove "\r".

Reply:
But, many place use only "\n", we need make the consistency here, either "\r\n" or "\n". From the C standard, the console driver should auto add "\r" before "\n" if the platform need, so it's better to use "\n".

Reply:
There are console drivers don't auto add "\r", prefer to keep "\r".

Reply:
Ok, I will change all single \n to \r\n to keep the consistency.